extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1246,7 +1243,16 @@ export class AmpStoryPage extends AMP.BaseElement {
    * @private
    */
   maybeStartAnimations_() {
-    this.animationManager_?.animateIn();
+    if (!this.animationManager_) {
+      return;
+    }
+    if (prefersReducedMotion(this.win)) {
```

Comment:
1. This should be handled in the `AnimationRunner` level.

2. I'm not sure `finish`ing an animation will work with `prefers-reduced-motion`. Please add an HTML test case using native animations/no AMP.

3. You need to block the display of the page until the `amp-animation` extension has loaded. (We previously did not need to do this for the first frame on presets, since it does go through the extension).

Reply:
3. +1
Making it render blocking would however be trivial, we can cross that bridge later if we need to, this PR already improves the experience drastically.

Reply:
> 1. If we do this, it will apply to all amp-animation and not just stories. This would require extra work and so far, the prefers-reduced-motion is only being respected in stories. We might break many AMP pages if we roll out this to all AMP instead of just stories.

No, I mean the AnimationRunner on Stories. https://github.com/ampproject/amphtml/blob/b1538b3c929edb6ae702543640d3445dc6f2b956/extensions/amp-story/1.0/animation.js#L103

> Finishing an animation works with prefers-reduced-motion perfectly, not sure if it's possible to create a test with it since it's a property of the OS (if I fake the media query, the browser will still behave as before)

Right, I don't mean a unit test. I mean a test case. A web page I can open on a browser manually.

> This is a balance between LCP vs CLS, and on the story side we're trying to improve LCP so much that it sounds counter intuitive to introduce a large delay on rendering because of slow animations (which in many cases are just minor items not necessary for the page to be consumed). @gmajoulet wdyt about this?

> Making it render blocking would however be trivial, we can cross that bridge later if we need to, this PR already improves the experience drastically.

Have you actually tested how this looks?

Reply:
As discussed offline, we don't need to block the rendering, so resolving this conversation